### DISCUSSION TITLE
Error deploying to Vercel

### DISCUSSION THREAD

**Discussion by fprl** [NathanAP/vue-google-maps-community-fork#23] [Q&A] [answered]
Hey there, is anyone having a problem with Vercel? on dev its working good but when I try to `yarn build` its giving me this error:
 
I'm using this package in an Astro app.

### On yarn dev
this appears in `yarn dev` but still works.
```
vue-google-maps-community-fork doesn't appear to be written in CJS, but also doesn't appear to be a valid ES module (i.e. it doesn't have "type": "module" or an .mjs extension for the entry point). Please contact the package author to fix. (x3)
```

### On yarn build
```
error Failed to parse /Users/myuser/me/Work/project/node_modules/vue-google-maps-community-fork/src/components/marker.vue as script:
```

## Fix
I can confirm the error was in vite, you can solve it by:

```
vite: {
    ssr: {
      noExternal: ['@fawmi/vue-google-maps'],
    },
}
```

Regards

**Comment by NathanAP** [comment, marked as answer]
I found [this post](https://github.com/FortAwesome/Font-Awesome/issues/18514) that seems related.

Is that the only fix or is there something in our side that we could do?

**Reply by fprl** [reply]
Hey @NathanAP , I edited my post and that was exactly the same fix I found :)

Thanks
